$OpenBSD$

Reduce the SMP proxy stress-test load for OpenBSD.
The upstream load is too heavy for the local test environment.

--- tests/SMPProxyTests.hs.orig
+++ tests/SMPProxyTests.hs
@@ -139,9 +139,15 @@
           agentViaProxyRetryNoSession
       describe "stress test 1k" $ do
         let deliver nAgents nMsgs = agentDeliverMessagesViaProxyConc (replicate nAgents [srv1]) (map bshow [1 :: Int .. nMsgs])
+#if defined(openbsd_HOST_OS)
+        it "2 agents, 50 messages" . oneServer $ deliver 2 50
+        it "5 agents, 10 pairs, 10 messages, N1" . oneServer . withNumCapabilities 1 $ deliver 5 10
+        it "5 agents, 10 pairs, 10 messages. N4" . oneServer . withNumCapabilities 4 $ deliver 5 10
+#else
         it "2 agents, 250 messages" . oneServer $ deliver 2 250
         it "5 agents, 10 pairs, 50 messages, N1" . oneServer . withNumCapabilities 1 $ deliver 5 50
         it "5 agents, 10 pairs, 50 messages. N4" . oneServer . withNumCapabilities 4 $ deliver 5 50
+#endif
       xdescribe "stress test 10k" $ do
         let deliver nAgents nMsgs = agentDeliverMessagesViaProxyConc (replicate nAgents [srv1]) (map bshow [1 :: Int .. nMsgs])
         it "25 agents, 300 pairs, 17 messages" . oneServer . withNumCapabilities 4 $ deliver 25 17
